.gitpod.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM gitpod/workspace-mysql
+
+USER root
+
+# Update APT Database
+### base ###
+RUN sudo apt-get update -q \
+    && sudo apt-get install -y php-dev
+
+# Install XDebug
+RUN wget http://xdebug.org/files/xdebug-2.9.1.tgz \
+    && tar -xvzf xdebug-2.9.1.tgz \
+    && cd xdebug-2.9.1 \
+    && phpize \
+    && ./configure \
+    && make \
+    && sudo mkdir -p /usr/lib/php/20190902 \
+    && sudo cp modules/xdebug.so /usr/lib/php/20190902 \
+    && sudo bash -c "echo -e '\nzend_extension = /usr/lib/php/20190902/xdebug.so\n[XDebug]\nxdebug.remote_enable = 1\nxdebug.remote_autostart = 1\n' >> /etc/php/7.4/cli/php.ini"
+
+# Install latest composer v2 release
+RUN curl -s https://getcomposer.org/installer | php \
```

Comment:
You can install composer with this one-line command (no sudo required since the current user is `root`):

```
curl -sSf https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer
```